pkg/controller/dynamodb/table/hooks.go

```diff
@@ -368,9 +389,66 @@ func isUpToDate(cr *svcapitypes.Table, resp *svcsdk.DescribeTableOutput) (bool,
 	case len(diffGlobalSecondaryIndexes(GenerateGlobalSecondaryIndexDescriptions(cr.Spec.ForProvider.GlobalSecondaryIndexes), resp.Table.GlobalSecondaryIndexes)) != 0:
 		return false, nil
 	}
+
+	return true, nil
+}
+
+func isPitrUpToDate(cr *svcapitypes.Table, pitrStatusBool bool) bool {
+	// it is not up to date when either point in time recovery is set and the state doesn't match the one on aws
+	// or it is unset and it is enabled on aws
+	return !((cr.Spec.ForProvider.PointInTimeRecoveryEnabled != nil && *cr.Spec.ForProvider.PointInTimeRecoveryEnabled != pitrStatusBool) ||
+		(cr.Spec.ForProvider.PointInTimeRecoveryEnabled == nil && pitrStatusBool))
+}
+
+func (e *updateClient) isUpToDate(cr *svcapitypes.Table, resp *svcsdk.DescribeTableOutput) (bool, error) {
+	// A table that's currently creating, deleting, or updating can't be
+	// updated, so we temporarily consider it to be up-to-date no matter
+	// what.
+	switch aws.StringValue(cr.Status.AtProvider.TableStatus) {
+	case string(svcapitypes.TableStatus_SDK_UPDATING), string(svcapitypes.TableStatus_SDK_CREATING), string(svcapitypes.TableStatus_SDK_DELETING):
+		return true, nil
+	}
+
+	// Similarly, a table that's currently updating its SSE status can't be
+	// updated, so we temporarily consider it to be up-to-date.
+	if cr.Status.AtProvider.SSEDescription != nil && aws.StringValue(cr.Status.AtProvider.SSEDescription.Status) == string(svcapitypes.SSEStatus_UPDATING) {
+		return true, nil
+	}
+
+	coreUpToDate, err := isCoreResourceUpToDate(cr, resp)
+	if err != nil {
+		return false, err
+	}
+	if !coreUpToDate {
+		return false, nil
+	}
+
+	// point in time recovery status
+	cbresult, err := e.client.DescribeContinuousBackups(&svcsdk.DescribeContinuousBackupsInput{
+		TableName: aws.String(cr.Name),
+	})
+	if err != nil {
+		return false, err
+	}
+	pitrStatus := cbresult.ContinuousBackupsDescription.PointInTimeRecoveryDescription.PointInTimeRecoveryStatus
+	pitrStatusBool := pitrStatusToBool(pitrStatus)
+
+	if !isPitrUpToDate(cr, pitrStatusBool) {
+		return false, nil
+	}
+
 	return true, nil
 }
 
+func pitrStatusToBool(pitrStatus *string) bool {
+	pitrStatusBool := false
+	if *pitrStatus == string(svcapitypes.PointInTimeRecoveryStatus_ENABLED) {
+		pitrStatusBool = true
+	}
+
+	return pitrStatusBool
```

Comment:
This can be simplified

```suggestion
        return pointer.StringDeref(pitrStatus, "") == string(svcapitypes.PointInTimeRecoveryStatus_ENABLED)
```

Reply:
Sure no worries :) Some maintainers like if it's split so i wasn't sure. Gonna apply this and squash. Thanks!

Reply:
nitpick applied and squashed